packages/design-system/yarn.lock

```diff
@@ -0,0 +1 @@
+../../yarn.lock
```

Comment:
Why do we need the lockfile here where it does not seems to be needed in the other packages in the monorepo ?

Reply:
Good question, TS really wanted it IIRC, @frassinier?

Reply:
It's for Security Scan purpose